data/core/editor/help.cfg

```diff
@@ -122,6 +122,63 @@ Named locations can be accessed from WML using the Standard Location Filter’s
+        # In English the text is “IO” for impassable and “UO” for unwalkable.
+        _ "editor^<bold>text='Movement Overlays'</bold>
+
+<img>src='terrain/grass/green.png~BLIT(terrain/impassable-editor.png~O(0.5))' align=here box=yes</img> <img>src='terrain/grass/green.png~BLIT(terrain/unwalkable-editor.png~O(0.5))' align=here box=yes</img> Impassable and Unwalkable
+
+While easily noticeable in the editor, these are invisible in the game, so the mixed terrains created by them look like the base terrain. They create a mixed terrain with the movement costs set to “impassable” or “unwalkable” respectively." + "
+
+" +
+        # po: The images here have text, while they could be translated I assume editor-only images won’t be.
+        # In English these images are the literal text “Castle overlay” and “Keep overlay”.
+        _ "editor^<bold>text='Castle Overlays'</bold>
+
+<img>src='terrain/grass/green.png~BLIT(terrain/castle/castle-overlay-editor.png~O(0.5))' align=here box=yes</img> <img>src='terrain/grass/green.png~BLIT(terrain/castle/keep-overlay-editor.png~O(0.5))' align=here box=yes</img>
+
+Adding either of these overlays to a passable hex allows units to be recruited onto a hex. The keep also allows a leader to recruit from there.
+
+These can be added to an impassable hex to connect a castle to a visually-separate keep through an impassable wall. It’s also possible to create a castle that seems to have grassland between the keep and towers, however this requires the connecting hexes to be occupied or blocked to prevent units being recruited onto them." + "
+
+" +
+        # po: The image here has text, while it could be translated I assume editor-only images won’t be.
+        # In English this image is the literal text “Village overlay”.
+        _ "editor^<bold>text='Village Overlay'</bold>
+
+<img>src='terrain/grass/green.png~BLIT(terrain/village/village-overlay-editor.png~O(0.5))' align=here box=yes</img>
+
+This turns any base terrain into a village, providing income and healing." + "
+
+" +
+        # po: The image is an “S” on a solid black background.
+        _ "editor^<bold>text='Fake Shroud'</bold>
+
+<img>src=terrain/void/shroud-editor.png align=left box=yes</img>
+
+Fake Shroud looks like an unexplored area, even in scenarios that have shroud disabled and even when the player’s units can see the hex."
+[/topic]
+
+[topic]
+    id=editor_deprecated_overlay
+    title= _ "Deprecated Terrain"
+    # po: The main reason for choosing “D” is that the hole in the middle of the letter makes it easier to see which terrain is underneath it. This uses a hardcoded image and doesn’t expect the image to be translated.
+    # The Xol and ^Efs terrains' help pages aren't given hyperlinks, because they both have hide_help enabled. Even when on a map with Xol on it, ctrl+t will show the hidden page but the ref still won’t link to it.
```

Comment:
I forgot that WML `# po:` hints need the `po:` on each line, not just the first one. The next force push fixes that, and consistently uses typographical apostrophes in this po comment, with no other changes.